specification/maps/data-plane/Creator/preview/2022-01-01-preview/dataset.json

```diff
@@ -0,0 +1,377 @@
+      "description": "The identifier for the dataset to query from.",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "AppendDatasetId": {
+      "name": "datasetId",
+      "type": "string",
+      "in": "query",
+      "description": "The ID for the dataset to append with. The dataset must originate from a previous dataset creation call that matches the datasetId",
+      "x-ms-parameter-location": "method"
+    },
+    "DatasetOperationId": {
+      "name": "operationId",
+      "type": "string",
+      "in": "path",
+      "description": "The ID to query the status for the dataset create/import request.",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    }
+  },
+  "paths": {
+    "/datasets": {
+      "post": {
+        "description": "**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\nCreator makes it possible to develop applications based on your private indoor map data using Azure Maps API and SDK. [This](https://docs.microsoft.com/azure/azure-maps/creator-indoor-maps) article introduces concepts and tools that apply to Azure Maps Creator.\n\n This API allows the caller to create a dataset from data that was uploaded to the Azure Maps Data Service and converted using the Azure Maps Conversion Service. You can also create a dataset directly using data uploaded to the Azure Maps Data Service, provided that the uploaded data is a zip of geojson files conforming to the [facility ontology](https://docs.microsoft.com/en-us/azure/azure-maps/creator-facility-ontology) format. \n\nYou can use this API in a scenario like uploading a DWG zip package for a building, converting the zip package using the Azure Maps Conversion Service, and creating a dataset from the converted zip package. Or you can upload a zip of geojson files, in the [facility ontology](https://docs.microsoft.com/en-us/azure/azure-maps/creator-facility-ontology) format and create a dataset from it. The created dataset can be used to create tilesets using the Azure Maps Tileset Service and can be queried via the Azure Maps WFS Service.\n\n### Submit Create Request\n\nTo create your dataset, you will use a `POST` request where the `conversionId` query parameter is an ID that represents the converted DWG zip package, the `udid` parameter is an ID representing the uploaded zip of geojsons, the `outputOntology` parameter is the name of ontology for the uploaded zip file (currently only supporting facility-2.0) the `datasetId` parameter will be the ID of a previously created dataset to append with the current dataset and, optionally, the `description` query parameter will contain a description (if description is not provided a default description will be given).\n\n**Note:** `outputOntology` must be provided with the `udid` parameter. Also, `conversionId` is mutually exclusive with `udid` and `outputOntology`.\n\nThe Create API is a [long-running request](https://aka.ms/am-creator-lrt-v2).",
```

Comment:
@brendansco - Added links to other services, however, are there any aka.ms links that we can add. Since the links are to a specific version of the service, they will need to be changed whenever we bump up versions of other services, which could be a tedious task.